prow/config/config_test.go

```diff
@@ -2373,12 +2384,18 @@ func TestSlackReporterValidation(t *testing.T) {
 			channel:        "my-channel",
 			reportTemplate: "{{ .Undef}}",
 		},
+		{
```

Comment:
Nit: It looks like you accidentally copied the testcase

Reply:
🤦‍♂